server/console_storage.go

```diff
@@ -298,7 +298,7 @@ func (s *ConsoleServer) ListStorage(ctx context.Context, in *console.ListStorage
 		var createTime pgtype.Timestamptz
 		var updateTime pgtype.Timestamptz
 
-		if err := rows.Scan(&o.Collection, &o.Key, &o.UserId, &o.Value, &o.Version, &o.PermissionRead, &o.PermissionWrite, &createTime, &updateTime); err != nil {
+		if err := rows.Scan(&o.Collection, &o.Key, &o.UserId, &o.Version, &o.PermissionRead, &o.PermissionWrite, &createTime, &updateTime); err != nil {
```

Comment:
💅  I think `console.proto` should be updated to reflect this change. Either removing the field since it only powers the UI Console AFAIK, or by marking it was deprecated.